www/src/content/docs/components/Otel-middleware.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Otel middleware
+description: description of otel middleware
+---
+
+The Hono [Otel middleware](https://github.com/fiberplane/fpx/tree/main/packages/client-library-otel) leverages [OpenTelemetry](https://opentelemetry.io/) to instrument applications and send traces to FPX Studio. It requires the use of [Hono](https://hono.dev/) web application framework.
+
+## Installation 
+To install the Otel middleware for your Hono application, run the following command in your project directory:
+```
+npm i @fiberplane/hono-otel@beta 
+``` 
+
+## Usage within the Hono application
+Below is an example of how to use the Otel middleware within a Hono application:
+```typescript
+import { Hono } from "hono";
+import { instrument } from "@fiberplane/hono-otel";
+
+const app = new Hono();
+
+app.get("/", (c) => c.text("Hello, Hono!"));
+
+export default instrument(app);
+```
+
+### Advanced usage: Custom spans
+The measure function from the Hono Otel middleware allows you to create custom spans for any function within your application, enabling more granular tracing.
```

Comment:
Question for discussion: Can/should we link to something like "what is a span" or "what is a trace"?

Reply:
what about this link form the Otel docs: https://opentelemetry.io/docs/concepts/signals/traces/

Reply:
I think we can link for now but ideally we'd explain those on our site, without having to link off elsewhere for the core explanations (linking for source and in-depth explorations ofc).